README.md

```diff
@@ -48,9 +48,12 @@
 ## Use It!
 
 ```bash
-git clone https://github.com/antfu/vite-starter && cd vite-starter && rm -rf .git && git init && pnpm i
+npx degit antfu/vitesse my-vitesse-app \
+cd my-vitesse-app \
+git init \
+pnpm i # If you don't have pnpm installed, run: npm install -g pnpm
 ```
 
 ## Why
 
-I have created several Vite apps recently. Setting the configs up is kinda the bottleneck for me to make the idea simply comes true in very a short time. So I made this starter template for myself to create apps more easily, along with some good practices that I have learned during making those apps. It's strongly opinionated, but feel free to tweak it or even maintains your own forks.
\ No newline at end of file
+I have created several Vite apps recently. Setting the configs up is kinda the bottleneck for me to make the idea simply comes true in very a short time. So I made this starter template for myself to create apps more easily, along with some good practices that I have learned during making those apps. It's strongly opinionated, but feel free to tweak it or even maintains your own forks.
```

Comment:
When editing README's directly on Github, always happening something at the end of file. I did'nt touch here. Maybe something about line endings :/

Reply:
No prob. I should blame my editor. Will have a look on if it can do it automatically for me😅